ci/environment-py34.yml

```diff
@@ -3,7 +3,7 @@ channels:
   - conda-forge
 dependencies:
   - python=3.4
-  - scipy
+  - scipy=0.19
```

Comment:
I think you may need to pin `0.19.0` explicitly (it seems `0.19.1` is being installed): https://travis-ci.org/spencerahill/aospy/jobs/301587193#L641

Reply:
Blarg.  Good catch, thanks.